src/models/changeinfos.js

```diff
@@ -0,0 +1,27 @@
+const {
```

Comment:
I see, that some of files with models are named in lowercase. That's incorrect, they should be in camelCase.
Use this [page](https://daily-dev-tips.com/posts/git-basics-help-my-case-sensitive-filename-changes-dont-commit/) to correctly rename these files and track them in git

Reply:
Okay, we decided to not do that.